app/api/auth.py

```diff
@@ -94,6 +94,8 @@ def login_user(provider, auth_code):
             'client_secret': provider_class.get_client_secret(),
             'code': auth_code
         }
+        if not payload['client_id'] or payload['client_secret']:
+            return make_response(jsonify(message="Facebook Login hasn't been configured"))
```

Comment:
Please paste what the response will look like. Also I think this will send 200 OK as a code which is definitely worse than 500

Reply:
I think you're right. But what code would you suggest me to return?

Reply:
501 Not Implemented

Reply:
Please paste the JSON response

Reply:
501 done. Will paste the response.